sdk/src/store.rs

```diff
@@ -1981,6 +1984,14 @@ impl Store {
         }
     }
 
+    /// check the input url to see if it is a supported remotes URI
+    pub fn is_valid_remote_url(url: &str) -> bool {
+        match url::Url::parse(url) {
+            Ok(u) => u.scheme().starts_with("http"),
```

Comment:
This could misfire with a bogus URL scheme such as "httpx". Not sure how that might be abused, but I'd prefer to see the specific test for "http" and "https."